lib/shared/bucketing-test-data/.gitignore

```diff
@@ -0,0 +1 @@
+json-data/*
```

Comment:
removing the json data from git because it has dates in it that are recomputed on every run and cause tests to fail if they aren't relative to the current date